src/lib/operators/table_scan/is_null_table_scan_impl.hpp

```diff
@@ -20,9 +20,15 @@ class IsNullTableScanImpl : public BaseSingleColumnTableScanImpl {
   void handle_value_column(const BaseValueColumn& base_column,
                            std::shared_ptr<ColumnVisitableContext> base_context) override;
 
+  void handle_dictionary_column(const BaseDeprecatedDictionaryColumn& base_column,
```

Comment:
Originally, these methods had different names, because there were no base classes. No the different names aren’t really necessary anymore. I would just remove all the prefixes and just call all methods the same: `handle_column`.

Reply:
Sounds good.